src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/entity/Referral.kt

```diff
@@ -40,6 +45,8 @@ data class Referral(
   @Column(name = "desired_outcome_id")
   var desiredOutcomeIDs: List<UUID>? = null,
   var completionDeadline: LocalDate? = null,
+
+  @NotNull val serviceUserCRN: String,
```

Comment:
Given `serviceUserCRN` is a required field, what do you think about moving it to the top?

Reply:
these three fields at the bottom are all sort of 'required', but postgres generates the other two. that's the reason for the grouping. can move to the top if you like though.